code/datums/gamemodes/gangwar.dm

```diff
@@ -236,26 +239,15 @@
 	potential_hot_zones = list()
 
 	for(var/area/A as area in world)
-		if(A.z != 1 || A.teleport_blocked || istype(A, /area/supply) || istype(A, /area/shuttle/) || A.name == "Space" || A.name == "Ocean")
+		if(A.z != Z_LEVEL_STATION || A.teleport_blocked || istype(A, /area/supply) || istype(A, /area/shuttle/) || A.name == "Space" || A.name == "Ocean")
```

Comment:
Why not just filter by station areas lol

Reply:
this big jumble of checks is in the original code, i'd assumed it was to cover some weird edge case with station areas 🤷